staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/validator.go

```diff
@@ -77,6 +80,24 @@ func (v *validator) Validate(ctx context.Context, versionedAttr *admission.Versi
 		f = *v.failPolicy
 	}
 
+	matchResults := v.celMatcher.Match(ctx, versionedAttr, versionedParams)
```

Comment:
related to the above question, is `celMatcher` always non-nil? should it be nil and should we skip this if we have no `matchConditions`?